Bottom: aa830536d8a2ad0dc2826412b291cf6171832ccd
Top:    4aedeff3fbdc7edda79bd9217b52850a061a12bf
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 22:10:43 +0000

Refresh of remove-debug-from-outline

---

diff --git a/src/polygon.c b/src/polygon.c
index 46101dc..2ba4f52 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -2243,19 +2243,18 @@ struct clip_outline_info {
 
 #define ROUTER_THICKNESS MIL_TO_COORD (10)
 //#define ROUTER_THICKNESS MIL_TO_COORD (0.1)
-#undef ROUTER_THICKNESS
-#define ROUTER_THICKNESS (line->Thickness)
+
 static int
 arc_outline_callback (const BoxType * b, void *cl)
 {
-  ArcType *line = (ArcType *)b;
+  ArcType *arc = (ArcType *)b;
   struct clip_outline_info *info = cl;
   POLYAREA *np, *res;
 
 #ifdef DEBUG_CIRCSEGS
-  if (!(np = ArcPoly (line, line->Thickness)))
+  if (!(np = ArcPoly (arc, arc->Thickness)))
 #else
-  if (!(np = ArcPoly (line, ROUTER_THICKNESS)))
+  if (!(np = ArcPoly (arc, ROUTER_THICKNESS)))
 #endif
     return 0;
